deploy/charts/crate-operator-crds/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: crate-operator-crds
+description: Crate Operator CRDs - Helm chart for installing and upgrading Custom Resource Definitions (CRDs) for the Crate Operator.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
What do you think about the version here - should we keep it in sync with the operator version? What is the best practice?

Reply:
idk what to think about it. If it's the same, it's easier to figure out which release is the good one. On the other side, if we want to change the chart or its templates we would have to change the operator version too.

Reply:
I would be ok with that, would rather the charts and the operator are a single distribution that share a version. But not sure what the best practice is. WDYT @WalBeh ?